src/cr/cube/cube.py

```diff
@@ -726,6 +737,7 @@ def valid_counts(self):
             .get("valid_count_unweighted", {})
             .get("data", [])
         )
+
```

Comment:
Accidental blank line? I don't think it's needed here, as the previous line only has a single character, so everything is pretty visible. I'm all for adding blank lines when they increase visibility though :)